### DISCUSSION TITLE
加强版宽屏在v2版本导入不了设置

### DISCUSSION THREAD

**Discussion by SpoonerCake** [the1812/Bilibili-Evolved#2792] [提问 / Q&A]
这是之前的代码，然后现在提示没有![image](https://user-images.githubusercontent.com/49234232/147715449-7fa15f62-222b-498d-a983-65dbb188270a.png)这个功能，现在能怎么导入呢

**Comment by the1812** [comment]
`bilibiliEvolved.installStyle` 改成 `bilibiliEvolved.pluginApis.installStyle`

**Reply by SpoonerCake** [reply]
今天加强版宽屏定位偏移了如图，视频向右拓展了但是左边没有动，通过滑动可以移回，请问怎么才能修复呢
<img width="1727" alt="QQ20220607-182250@2x" src="https://user-images.githubusercontent.com/49234232/172359339-59cd0b1a-194a-401b-966e-288a6c49c55f.png">
